bigbluebutton-html5/imports/ui/components/app/container.jsx

```diff
@@ -150,7 +150,16 @@ export default injectIntl(withModalMounter(withTracker(({ intl, baseControls })
       setTimeout(() => {
         const queryCurrentUser = Users.find({ userId: Auth.userID, meetingId: Auth.meetingID });
         if (queryCurrentUser.count() === 0) {
-          endMeeting(403, userData.ejectedReason || null);
+          if (userData.ejected) {
+            endMeeting('403', userData.ejectedReason);
```

Comment:
If you want to change the codes to String instead of Integer.
Make sure it will be read as String in all places!

I think the `intlMessages` keys in `ErrorScreen` and `MeetingEnded` are being set as Integer!
And this way it would not work when using `intlMessages[code]` cause keys are Ints and code will be a String.

Please ignore if you are already handling this cases.